connector/aws/s3/s3.go

```diff
@@ -157,4 +199,52 @@ func (s3Client *S3Client) New() (err error) {
 	return
 }
 
+func (s3Client *S3Client) IsFileExists(key string) (exists bool) {
```

Comment:
In this method, the naked return value is not required, true and false can be returned directly instead of defining `exists` variable.